### PR TITLE
make top-level README point to feature_extractor/README + fix README …

### PR DESCRIPTION
**Pull request by samihaija**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -457,6 +458,13 @@ This sample code contains implementations of the models given in the
                              average-pooling on the fly over frame-level
                              features rather than using pre-aggregated features.
 
+## Create Your Own Dataset Files
+You can create your dataset files from your own videos. Our
+[feature extractor](./blob/master/feature_extractor) code creates `tfrecord`
+files, identical to our dataset files. You can use our starter code on the
+`tfrecord` files output by the feature extractor. In addition, you to fine-tune
```
Comment (LeegleechN) [inline README.md]: typo

File: README.md [hunk 2]
```diff
@@ -457,6 +458,13 @@ This sample code contains implementations of the models given in the
                              average-pooling on the fly over frame-level
                              features rather than using pre-aggregated features.
 
+## Create Your Own Dataset Files
+You can create your dataset files from your own videos. Our
+[feature extractor](./blob/master/feature_extractor) code creates `tfrecord`
```
Comment (LeegleechN) [inline README.md]: This link doesn't work for me.

Comment (samihaija) [reply README.md]: It will work, once this is checked in, I think

File: README.md (hunk 1 above)
Comment (samihaija) [reply README.md]: fixed

**Pull request merged**